TestAssets/TestProjects/TestAppWithLibrary/TestApp/project.json

```diff
@@ -9,11 +9,9 @@
       "target": "project",
       "version": "1.0.0-*"
     },
-    "NETStandard.Library": "1.5.0-rc2-24018"
+    "Microsoft.NETCore.App": "1.0.0-rc2-*"
```

Comment:
We restore these against the package that we built earlier in the build. 

The version is determined at build time.